pkg/authorizer/server/types.go

```diff
@@ -1,21 +1,12 @@
 package server
 
-type TokenRequest struct {
-	APIVersion string `json:"api_version"`
-
-	AuthorizationToken string `json:"authorization_token"`
+type ClusterRegistration struct {
```

Comment:
it looks like this type is never used outside of this package. can we make it unexported?

Reply:
yes, we can definitely make this unexported. In the future this should also get proper godoc, or even be auto-generated from the OpenAPI spec.